src/core_write.cpp

```diff
@@ -162,6 +172,7 @@ void TxToUniv(const CTransaction& tx, const uint256& hashBlock, UniValue& entry,
     entry.pushKV("size", (int)::GetSerializeSize(tx, SER_NETWORK, PROTOCOL_VERSION));
     entry.pushKV("vsize", (GetTransactionWeight(tx) + WITNESS_SCALE_FACTOR - 1) / WITNESS_SCALE_FACTOR);
     entry.pushKV("locktime", (int64_t)tx.nLockTime);
+    entry.pushKV("bip125-replaceable", SignalsOptInRBF(tx));
```

Comment:
`SignalsOptInRBF` is *not* bip125-replacable.

* Confirmed transaction can never be replaced in the mempool
* Unconfirmed transactions that depend on a mempool-transaction that signals opt in rbf can be replaced

Reply:
Would you suggest to rename the flag to something like `signals-rbf`?

Reply:
If we do this I'd say rename it to `signals-rbf` or such, which is also the name of the function already. And not make the raw transactions utility function decoding functionality depend on the mempool.

Reply:
In which case I tend to Concept NACK. Just returning if the flag is set contradicts the bip125-replacability rules.

Reply:
Sorry to hijack.

I had considered adding a flag like this to the verbose output of `getrawmempool`. Assuming this idea were supported, based on your feedback @MarcoFalke - is it fair to say that you think it makes more sense to list a mempool transaction as `replaceable` (covering both explicit signaling and implicit based on ancestors) rather than just `signals-rbf` (just the explicit case)? 

Do you think it would be appropriate to list that in `getrawmempool`?